### PR TITLE
Add codergrl arad recommendations

### PR DESCRIPTION
**Pull request by ghost**
#31

### CODE REVIEW

**Comment by ghost** [comment]
closes #31 

hey @asunshinehill , how does this look?

cc: @beardofedu

File: codergrl-arad.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#Arad, Romania
```
Comment (KevinButlerEsri) [inline codergrl-arad.md:1]: Sounds like a nice place.

File: codergrl-arad.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+###Places to Eat
+- My mom's place
```
Comment (asunshinehill) [inline codergrl-arad.md:4]: Is she a good cook? :hamburger:

File: codergrl-arad.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+- I never eat out when I'm there
+
+###Places to Go
+- National Theater
+- Cathedrals Downtown
```
Comment (KevinButlerEsri) [inline codergrl-arad.md:10]: What a great recommendation.

**Comment by KevinButlerEsri** [comment]
Very nice to have an international city in our project.

File: codergrl-arad.md (hunk 2 above)
Comment (ghost) [reply codergrl-arad.md:4]: Very!

**Comment by ghost** [comment]
@asunshinehill  is this good?

**Pull request merged**